active-record/etc/active-record.urm.puml

```diff
@@ -0,0 +1,40 @@
+@startuml
```

Comment:
You need to upload the image generated by this yourself. There is info about it on the wiki.